donate.html

```diff
@@ -68,7 +68,37 @@
 <div class="container-fluid">
   <!-- su'ad -->
 
-  <!-- jules -->
+  <!-- donation amount suggestions and action -->
+  <section class="dono-impact">
+    <h1>Your Donation's Impact</h1>
+    <br>
+    <h1>$20</h1>
+        <p>The cost to provide electricity and internet to a hacker, so they can stay charged up and logged in all weekend</p>
+    <h1>$10</h1>
+        <p>The cost of each meal we provide to hackers at Technica, keeping them fueled and properly fed</p>
+    <h1>$5</h1>
+        <p>The cost per hacker to keep Emergency Medical Services and Security Personnel on hand throughout Technica weekend, just in case</p>
+    <br>
+    <a target="_blank" href="https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=STSQQD9BAPZSJ&source=url" class="dono-button">Donate</a>
+  </section>
+
+  <!-- footer with contact and socials -->
+  <section class="dono-questions">
+    <h2>Questions about donating? Please email <b><a href="mailto:finance@gotechnica.org">finance@gotechnica.org</a></b></h2>
```

Comment:
Although this is consistent with the ambassadors page, the paragraph tag would be more suitable for this bottom text following the wireframe (imo the smaller text would be more cohesive with the donation page overall)

Reply:
lets also center this!